apollo-router/src/plugins/connectors/handle_responses.rs

```diff
@@ -207,9 +234,8 @@ impl RawResponse {
                 };
 
                 // Now we can create the error object using either the default message or the message calculated by the JSONSelection
-                let mut error = graphql::Error::builder()
-                    .message(message)
-                    .path(Path::from_response_key(&key));
+                let mut error = RuntimeError::new(message, &key);
+                error.subgraph_name = Some(connector.id.subgraph_name.clone());
```

Comment:
```suggestion
                error.subgraph_name = Some(connector.id.subgraph_name.clone());
                error.coordinate = Some(connector.id.coordinate().into());
```
Or maybe we don't want it in this case for a specific reason?

Wondering if `::new` should take in an optional Connector? Or store an `Arc<Connector>` and spit out subgraph name / coordinate on demand if we're worried about runtime cost?